Source/Utils/VerifyScript.swift

```diff
@@ -17,7 +17,9 @@ final class VerifyScript {
     }
 
     private init() {
```

Comment:
Can we get rid of this and signal a proper compile error?

Reply:
Sure. Will delete this class as it’s not intended to be used atm.

Reply:
@xxuejie Deleted the VerifyScript.